services/appflowy-collaborate/src/shared_state.rs

```diff
@@ -0,0 +1,74 @@
+use crate::error::RealtimeError;
+use futures_util::StreamExt;
+use redis::{pipe, AsyncCommands, AsyncIter};
+
+#[derive(Clone)]
+pub struct RealtimeSharedState {
+  redis_conn_manager: redis::aio::ConnectionManager,
+}
+
+impl RealtimeSharedState {
+  pub fn new(redis_conn_manager: redis::aio::ConnectionManager) -> Self {
+    Self { redis_conn_manager }
+  }
+  pub async fn add_connected_user(&self, uid: i64, device_id: &str) -> Result<(), RealtimeError> {
+    let mut conn = self.redis_conn_manager.clone();
+    let key = realtime_shared_state_cache_key(&uid, device_id);
+    conn
+      .set_ex(key, "1", 60 * 60 * 3)
```

Comment:
this is ok. just wondering if there's a better default value for this?

Reply:
I can use timestamp instead of "1"